src/stylesheets/app.scss

```diff
@@ -0,0 +1,276 @@
+      top: -40%;
+      right: -35%;
+    }
+  }
+
+  #purple-pills{
+    position: absolute;
+    right: 10%;
+    width: 12%;
+    @media (max-width: 600px){
+      width: 35%;
+      bottom: -30%;
+      right: -22%;
+      z-index: 4;
+    }
+  }
+}
+
+/***** Pillbar styles *****/
+
+.pillbar{
+  color: #d2e7e2;
+  background-image: linear-gradient(0deg, #e95e67 75%, #194e67 25%);
+  background-image: -webkit-linear-gradient(0deg, #e95e67 75%, #194e67 25%);
+  -moz-box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  -webkit-box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  box-shadow: inset 2px -15px 2px rgba(0, 0, 0, 0.1);
+  height: 35px;
+  border-radius: 70px;
+  display: flex;
+  width: 40%;
+  margin: 5% auto;
+  font-family: 'geomanist';
+  @media (max-width: 600px){
+    display: none;
+  }
+  a{
+    color: #d2e7e2;
+    text-decoration: none;
+  }
+  ul{
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    width: 100%;
+    margin: auto;
+    text-transform: uppercase;
+    font-size: 11px;
+  }
+}
+
+.box{
+  max-width: 70%;
+  background: #ffffff;
+  border-radius: 4px;
+  margin: auto;
+  padding: 2% 5%;
+  height: 100%;
+  @media (max-width: 600px){
+    max-width: 100%;
+    padding-bottom: 10%;
+  }
+}
+
+
+/**** First box ****/
+#box-one{
+  position: relative;
+}
+
+#main-thought{
+  font-family: 'Merriweather';
+  font-size: 1.7em;
+  line-height: 1.5em;
+  @media (max-width: 600px){
+    font-size: 1.4em;
+  }
+}
+
+.quote{
+  text-align: center;
+  width: 60%;
+  margin-left: 38%;
+}
+
+#quote-text{
+  font-family: 'Merriweather';
+  color: $florence-orange;
+  @media (max-width: 600px){
+    text-align: left;
+  }
+}
+
+.attribution{
+  font-family: 'geomanist-medium';
+  font-size: 0.75em;
+  @media (max-width: 600px){
+    text-align: left;
+  }
+}
+
+#pill-bottle{
+  position: absolute;
+  width: 17%;
+  bottom: -25%;
+  left: 0;
+  z-index: 3;
+  @media (max-width: 600px){
+    width: 50%;
+    left: -20%;
+    bottom: 7%;
+    max-height: 200px;
+  }
+}
+
+#pill-bottle2{
+  position: absolute;
+  width: 13%;
+  bottom: -15%;
+  left: 10%;
+  z-index: 1;
+  @media (max-width: 600px){
+    width: 30%;
+    bottom: 12%;
+    left: 3%;
+  }
+}
+
+#pill-bottle3{
+  position: absolute;
+  width: 19%;
+  bottom: -15%;
+  left: -10%;
+  z-index: 0;
+  @media (max-width: 600px){
+    display: none;
+  }
+}
+
+/*------- Key Factors -----------*/
+#key-factors {
+  color: #504949;;
+  margin-top: 10%;
+  h3{
+    color: $florence-orange;
+    text-transform: uppercase;
+    font-size: 0.8em;
+    margin-bottom: -8px;
+  }
+}
+.section-title{
+  text-transform: uppercase;
+  width: 100%;
+  text-align: center;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  color: $florence-blue;
+}
+.section-intro{
+  font-family: 'Merriweather';
+  width: 47%;
+  text-align: center;
+  margin: auto;
+  color: $florence-blue;
+}
+#body{
+  display: block;
+  width: 20%;
```

Comment:
i see you using width and margins in percentages frequently. this oftentimes creates some bad layout problems between responsive sizes. i recommend using several breakpoints and setting width to 100% with a `max-width` or `min-width` depending on the scenario to prevent it from getting too squished or too wide on certain screens.